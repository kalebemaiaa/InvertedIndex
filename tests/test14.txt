QueryAndL(QueryOr(QueryAnd(QueryWord("ministro"), QueryWord("secretário")), QueryOr(QueryWord("presidente"), QueryWord("primeiro-ministro"))), QueryAndS(QueryOr(QueryWord("diplomata"), QueryWord("embaixador")), QueryAnd(QueryWord("assessor"), QueryOr(QueryWord("deputado"), QueryWord("senador")))))
1177.raw
6010.raw
3080.raw
1135.raw
9013.raw
803.raw
1483.raw
5906.raw
3181.raw
6408.raw
243.raw
8803.raw
10867.raw
2913.raw
11063.raw
2778.raw
3559.raw
2610.raw
1585.raw
2701.raw
11272.raw
2618.raw
1501.raw
10878.raw
2200.raw
8806.raw
11405.raw
6072.raw
5142.raw
4664.raw
2324.raw
5719.raw
1176.raw
11696.raw
10953.raw
1968.raw
858.raw
6001.raw
334.raw
5004.raw
6011.raw
11704.raw
5473.raw
5074.raw
3314.raw
3162.raw
4405.raw
1037.raw
777.raw
11255.raw
10808.raw
1736.raw
1436.raw
6040.raw
11786.raw
3908.raw
8877.raw
5235.raw
4357.raw
3752.raw
2976.raw
11495.raw
5601.raw
1977.raw
16.raw
2072.raw
11574.raw
11022.raw
1605.raw
2950.raw
4581.raw
5738.raw
4462.raw
3226.raw
11133.raw
11493.raw
6054.raw
771.raw
3542.raw
4266.raw
3813.raw
1079.raw
4529.raw
3659.raw
4102.raw
1631.raw
5171.raw
11740.raw
20.raw
3019.raw
1009.raw
4401.raw
2454.raw
1621.raw
407.raw
2926.raw
1224.raw
4771.raw
480.raw
11303.raw
3719.raw
2511.raw
1117.raw
951.raw
3739.raw
3873.raw
3599.raw
568.raw
360.raw
3948.raw
10818.raw
2733.raw
6324.raw
3322.raw
10992.raw
4557.raw
2707.raw
2606.raw
1718.raw
5385.raw
10789.raw
2486.raw
5695.raw
11169.raw
2395.raw
1626.raw
5278.raw
4716.raw
4501.raw
1144.raw
5567.raw
4155.raw
5294.raw
5457.raw
2303.raw
11661.raw
10999.raw
2450.raw
3887.raw
2349.raw
5883.raw
461.raw
11979.raw
6006.raw
5623.raw
4920.raw
3282.raw
50.raw
3385.raw
4528.raw
12033.raw
5868.raw
11674.raw
11135.raw
5152.raw
892.raw
3230.raw
10943.raw
2014.raw
1331.raw
10994.raw
5717.raw
1314.raw
6417.raw
10980.raw
2179.raw
8825.raw
2138.raw
366.raw
2791.raw
5897.raw
3865.raw
3347.raw
1716.raw
10940.raw
5430.raw
4641.raw
1252.raw
1961.raw
2485.raw
6241.raw
5638.raw
6269.raw
4706.raw
230.raw
1864.raw
11593.raw
5779.raw
2567.raw
3077.raw
656.raw
2803.raw
5355.raw
3098.raw
695.raw
974.raw
2705.raw
2352.raw
1220.raw
3515.raw
