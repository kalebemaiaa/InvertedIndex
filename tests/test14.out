2567
5430
